### DISCUSSION TITLE
Will tft.GetTouch eventually work on the LilyGo t-display touch ESP32S3 board version

### DISCUSSION THREAD

**Discussion by reflectingme** [Bodmer/TFT_eSPI#2454] [Compatible displays and setup files]
Can't seem to get any of the touch functionality or widgets working with my LilyGo T-Display Touch board within this library.  Have installed TFT_widgets as well, but not sure if this touch version of the board is compatible.  

Any pointers please as I'd love to use the excellent TFT-eSPI.h library with the touch functionality instead of lvgl.

**Comment by Bodmer** [comment]
I don't have much information on that display. It might use a GT911 touch controller so this library can be used for the touch functions and the example code integrated into a TFT_eSPI sketch:
https://github.com/Xinyuan-LilyGO/T-Display-S3/tree/main/lib/TouchLib

Get the example running first.

**Comment by reflectingme** [comment]
Wow sir... thank you for such a prompt response and at such a late hour too.  Really, you are very much appreciated.  Thank you.
